pr_watch/migrations/0014_auto_20210315_1246.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.12 on 2021-03-15 12:46
```

Comment:
@0x29a, nit: name the migration file properly to indicate the change that it applies.